webapp/src/Entity/User.php

```diff
@@ -436,22 +436,21 @@ public function getRoleList(): array
     #[Serializer\Type('string')]
     public function getType(): ?string
     {
-        // Types allowed by the CCS Specs Contest API in order of most permissions to least.
+        // Types suggested by the CCS Specs Contest API in order of most permissions to least.
         // Either key=>value where key is the DOMjudge role and value is the API account type or
         // only value, where both the DOMjudge role and API type are the same.
-        $allowedTypes = ['admin', 'api_writer' => 'admin', 'api_reader' => 'admin',
-                         'jury' => 'judge', 'api_source_reader' => 'judge',
-                         'team'];
-        foreach ($allowedTypes as $role => $allowedType) {
+        $mappedTypes = ['admin', 'api_writer' => 'admin', 'api_reader' => 'admin',
+                        'jury' => 'judge', 'api_source_reader' => 'judge', 'team'];
```

Comment:
```suggestion
                        'jury' => 'judge', 'api_source_reader' => 'judge',
                        'team'];
```
The newline was there with a reason, to show all things mapped to admin, all things to judge and all things to teams.

Reply:
fixed